src/main/webapp/app/exam/manage/exercise-groups/exercise-groups.component.html

```diff
@@ -16,18 +16,18 @@ <h4 class="mb-0">{{ 'artemisApp.examManagement.exerciseGroups' | artemisTranslat
 <div class="mt-3 text-end" id="number-groups" style="font-size: 0.85rem">
     {{ 'artemisApp.examManagement.numberOfExerciseGroups' | artemisTranslate }} {{ exerciseGroups ? exerciseGroups.length : 0 }}
 </div>
-<div *ngFor="let exerciseGroup of exerciseGroups; let i = index" class="exercise-group mt-3">
+<div *ngFor="let exerciseGroup of exerciseGroups; let i = index" class="exercise-group mt-3" id="{{ 'group-' + i }}">
     <div class="d-flex p-3 header flex-wrap">
         <div class="me-3 d-flex justify-content-center" style="align-items: center">
-            <a (click)="moveUp(i)" [class.disabled]="i == 0" class="border-0 p-0 me-1 bg-transparent">
+            <a (click)="moveUp(i)" [class.disabled]="i === 0" class="border-0 p-0 me-1 bg-transparent">
                 <fa-icon [icon]="faAngleUp" style="font-size: 1.3rem"></fa-icon>
             </a>
             <a (click)="moveDown(i)" [class.disabled]="i >= exerciseGroups!.length - 1" class="border-0 p-0 bg-transparent">
                 <fa-icon [icon]="faAngleDown" style="font-size: 1.3rem"></fa-icon>
             </a>
         </div>
         <div class="d-flex flex-column justify-content-center">
-            <h5 class="font-weight-bold mb-0">{{ exerciseGroup.title }}</h5>
+            <h5 class="group-title font-weight-bold mb-0">{{ exerciseGroup.title }}</h5>
```

Comment:
I think if these are for element selection data-cy is better suited as others might remove useless css classes